src/gt4py/next/iterator/ir.py

```diff
@@ -13,7 +13,7 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Let's add a TODO to remove `itir.FencilDefinition`, `itir.Program` so that the intermediate state is clear.